app/forms/network-interface-edit.tsx

```diff
@@ -107,10 +107,16 @@ export function EditNetworkInterfaceForm({
                 submitTransitIp()
               }
             }}
+            validate={(value) => {
+              const result = validateIpNet(value)
+              if (result.type === 'error') return result.message
+            }}
```

Comment:
I'm thinking I might want to split `validateIpNet` into `parseIpNet` and `validateIpNet`, and the latter just returns a string or undefined so it could go here simply as `validate={validateIpNet}`